Light Vox Engine/Graphics/FrameResource.cpp

```diff
@@ -14,7 +13,7 @@ FrameResource::FrameResource(
     ID3D12DescriptorHeap * rtvHeap,
     ID3D12DescriptorHeap * cbvSrvHeap,
     D3D12_VIEWPORT * viewport,
-    UINT frameResourceIndex )
+    uint32_t frameResourceIndex )
```

Comment:
Again, I think it would be helpful to have some kind of using statement here for the index. Something that is more descriptive in this context ,like `using FrameResource_ID = uint32_t;`

Reply:
it's just an index i don't think we need to do a using for this